torchbenchmark/models/DALLE2_pytorch/__init__.py

```diff
@@ -0,0 +1,116 @@
+import argparse
+import random
+import torch
+
+import numpy as np
+
+from torchbenchmark.util.env_check import set_random_seed
+
+from dalle2_pytorch import DALLE2, Unet, Decoder, DiffusionPriorNetwork, DiffusionPrior, OpenAIClipAdapter
+import typing
+
+torch.backends.cudnn.deterministic = False
+torch.backends.cudnn.benchmark = False
+from pathlib import Path
+from ...util.model import BenchmarkModel
+from torchbenchmark.tasks import COMPUTER_VISION
+
+import io
+
+
+class Model(BenchmarkModel):
+    task = COMPUTER_VISION.GENERATION
+    DEFAULT_TRAIN_BSIZE = 4
+    DEFAULT_EVAL_BSIZE = 4
+
+    def __init__(self, test, device, batch_size=None, jit=False, extra_args=[]):
+        super().__init__(test=test, device=device, jit=jit, batch_size=batch_size, extra_args=extra_args)
+
+        if device == "cpu":
+            raise NotImplementedError("DALL-E 2 Not Supported on CPU")
+    
+        self.clip = OpenAIClipAdapter().cuda()
```

Comment:
We still suggest to use `.to(self.device)` here instead of `cuda()`, since there could be other devices other than `cpu` or `cuda` in the future (e.g. Apple or AMD GPU)